src/client/helpers/apiHelpers.js

```diff
@@ -110,8 +110,9 @@ export const isWalletTransaction = actionType =>
   actionType === accountHistoryConstants.DELEGATE_VESTING_SHARES ||
   actionType === accountHistoryConstants.CLAIM_REWARD_BALANCE;
 
-export const getLookupAccountNames = (name, limit = 5) =>
-  SteemAPI.sendAsync('lookup_accounts', [name, limit]);
+export const getAccountReputation = (name, limit = 5) =>
```

Comment:
I think it's safe to request for 20 accounts by default, that should give better results

Reply:
yeah ill update it to request 20